piicatcher/scanner.py

```diff
@@ -65,7 +65,10 @@ def scan(self, text):
 
 class ColumnNameScanner(Scanner):
     def __init__(self, exclude_regex=()):
```

Comment:
Why empty tuple btw? Forgot to point it out. Just make None as default?

Reply:
You would do empty list if you want to iterate over it

Reply:
Yup can make it `None` as default.